target/sim/vsim/start.cheshire_soc.tcl

```diff
@@ -12,7 +12,7 @@ set TESTBENCH tb_cheshire_soc
 # Set voptargs only if not already set to make overridable.
 # Default on fast simulation flags.
 if {![info exists VOPTARGS]} {
-    set VOPTARGS "-O5 +acc=p+tb_cheshire_soc. +noacc=p+cheshire_soc. +acc=r+stream_xbar"
+    set VOPTARGS "-O5 +acc=p+tb_cheshire_soc. +acc=p+cheshire_soc. +acc=r+stream_xbar"
```

Comment:
Please do not upstream these changes, keep them local or set `VOPTARGS` when locally running the simulation :) let the default as is now